test/param_types_test.rb

```diff
@@ -28,6 +24,12 @@ class Flappi::VersionsTest < MiniTest::Test
         assert @param_types_test.validate_param(0, 'BOOLEAN')
       end
 
+      should 'reject empty param' do
+        assert @param_types_test.validate_param('', 'Wibble')
```

Comment:
This is the test that we accept an empty string as a param. I'd restore the above test and put it there, currently it looks like we reject empty, which is wrong.